src/components/validation-form/index.tsx

```diff
@@ -0,0 +1,190 @@
+/* eslint-disable no-console */
+import React, { memo, FC, useState, FormEventHandler } from 'react';
+
+import { compose } from 'redux';
+
+import Button from '@fellesdatakatalog/button';
+
+import { ExpansionPanelBody, ExpansionPanelHead } from '../expansion-panel';
+import GraphFieldset from '../graph-fieldset';
+import Translation from '../translation';
+
+import SC from './styled';
+
+import type { ShapesCollection, ValidationRequest } from '../../types';
+import { InputType } from '../../types/enums';
+import Checkbox, { LabelPosition } from '../checkbox';
+
+interface ExternalProps {
+  dataGraph?: File | string;
+  shapesGraph?: File | string;
+  shapesCollection: ShapesCollection | null;
```

Comment:
Gjentagende props. Kanskje denne bør wrappes med with-validator i stedet for page?